models.py

```diff
@@ -0,0 +1,28 @@
+import peewee as pw
+
+
+db = pw.PostgresqlDatabase(
+    'postgres',
+    user='postgres',
+    password='postgres',
+    host='db',
+    port='5432'
+)
+
+
+class BaseModel(pw.Model):
+    class Meta:
+        database = db
```

Comment:
I think this is a base plug for future models. Please reflect this in pull request message